queries.php

```diff
@@ -51,14 +51,14 @@
 
 <script type="text/javascript">
     $(document).ready(function() {
-        tableApi = $('#all-queries').DataTable( {
-            "rowCallback": function( row, data, index ){
-            	if (data[4] == "Pi-holed") {
-            		$(row).css('color','red')
-            	}
-            	else{
-            		$(row).css('color','green')
-            	}
+        tableApi = $('#all-queries').DataTable({
+            "rowCallback": function (row, data, index ) {
```

Comment:
Just personal preference, but can you remove the space between function and the parentheses? (Same for for loops and such)